zokrates_core/src/standard.rs

```diff
@@ -1,6 +1,6 @@
 use crate::flat_absy::{FlatExpression, FlatExpressionList, FlatFunction, FlatStatement};
 use crate::flat_absy::{FlatParameter, FlatVariable};
-use crate::helpers::{DirectiveStatement, Helper, RustHelper};
+use crate::solvers::{FlatDirective, solver, Rustsolver};
```

Comment:
Shouldn't `Rustsolver` be CamelCase, i.e., `RustSolver`?

Reply:
Good catch, fixing this